tests/Toolkit/StrTest.php

```diff
@@ -139,6 +140,14 @@ public function testDate()
         $this->assertSame($time, Str::date($time));
         $this->assertSame('29.01.2020', Str::date($time, 'd.m.Y'));
 
+        // `intl` handler
+        $formatter = new IntlDateFormatter(null, IntlDateFormatter::SHORT, IntlDateFormatter::NONE);
+        $this->assertSame($time, Str::date($time, null, 'intl'));
+        $this->assertSame('1/29/20', Str::date($time, 'M/d/yy', 'intl'));
+        $this->assertSame('1/29/20', Str::date($time, $formatter));
+        $this->assertSame('1/29/20', Str::date($time, [IntlDateFormatter::SHORT , IntlDateFormatter::NONE]));
```

Comment:
Can you make the tests play with the format a bit more, e.g. also include the time in the format of the `$formatter` object and in another way in the string? This way we have a higher chance of catching regressions down the road.